### PR TITLE
 Eveodd.java #1467

### LINKED ISSUE #1467: number of even and odd elements in an array
I would like to do that

### PR DESCRIPTION
**Pull request by Anushka-Rajvanshi**
code to find even,odd digits and their count in an array.

Fixes #1467 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- code added to find even, odd numbers and their count in an array.
- 

#### Languages Used:
- Java.
- 

#### Files Added:

[Eveodd.docx](https://github.com/jainaman224/Algo_Ds_Notes/files/4282784/Eveodd.docx)

- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: 1-D_Array/Java/Eveodd.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import java.util.*;
+class Eveodd
+{
+ public static void main(String args[])
```
[the remark this answers is not in the document]
Comment (jainaman224) [reply 1-D_Array/Java/Eveodd.java]: Indent the file properly.

**Comment by Anushka-Rajvanshi** [comment]
Sorry for the inconevience. I have made the changes you requested. Please review. @jainaman224

**Comment by Anushka-Rajvanshi** [comment]
Sorry for the inconvenience. I have made the changes you requested. Please review @jainaman224

File: 1-D_Array/Java/Eveodd.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import java.util.* ;
+class Eveodd
```
Comment (jainaman224) [inline 1-D_Array/Java/Eveodd.java]: Add a new line above this.

File: 1-D_Array/Java/Eveodd.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            if((arr[i] % 2) != 0)
+            {
+                o++ ;
+                System.out.println(arr[i] + " ");
+            }
+        }
+        System.out.println("Number of even terms in the array is:" + e +  "\nAnd number of odd terms is:" +o);
+    }
+}
+//sample output:
```
Comment (jainaman224) [inline 1-D_Array/Java/Eveodd.java]: Add a new line above this.

File: 1-D_Array/Java/Eveodd.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+/*Enter number of elemnts in array:6
```
Comment (jainaman224) [inline 1-D_Array/Java/Eveodd.java]: Self review your PR. There is a typo in this this.

Comment (Anushka-Rajvanshi) [reply 1-D_Array/Java/Eveodd.java]: Sorry sir. Now should I make a  separate PR to submit the updated code? @jainaman224

**Comment by jainaman224** [comment]
You are Raksha right?

**Comment by Anushka-Rajvanshi** [comment]
No sir, I am Anushka.

**Comment by Anushka-Rajvanshi** [comment]
No sir, I am Anushka. @jainaman224

File: 1-D_Array/Java/Eveodd.java (hunk 4 above)
Comment (jainaman224) [reply 1-D_Array/Java/Eveodd.java]: No it is ok

**Comment by Anushka-Rajvanshi** [comment]
You reverted the changes after merging it . So, do I have to submit another PR after updating the code. @jainaman224

**Comment by jainaman224** [comment]
No

**Comment by jainaman224** [comment]
Fixed my mistake.

**Pull request merged**